opencti-platform/opencti-graphql/src/database/middleware.js

```diff
@@ -515,8 +516,11 @@ export const storeLoadByIdWithRefs = async (context, user, id, opts = {}) => {
   const elements = await storeLoadByIdsWithRefs(context, user, [id], opts);
   return elements.length > 0 ? R.head(elements) : null;
 };
-export const stixLoadById = async (context, user, id, opts = {}) => {
+export const stixLoadById = async (context, user, id, opts = {}, stixVersion = StixVersion.Stix_2_1) => {
```

Comment:
version must be setup before the opts if mandatory and inside the opts if optional but not after